src/Updater/DeptracStrategy.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace SensioLabs\Deptrac\Updater;
+
+use Humbug\SelfUpdate\Strategy\ShaStrategy;
+
+class DeptracStrategy extends ShaStrategy
+{
+    const PHAR_URL = 'http://get.sensiolabs.de/deptrac.phar';
```

Comment:
please dont use CONST (because const is "public"), i am fine with inlining this or private static :)